pkg/transform/transform.go

```diff
@@ -296,6 +307,30 @@ func (r *relLinkTransformer) TransformDestination(ctx mdformatter.SourceContext,
 		}
 	}
 
+	// Non md or image relative link, so needs link to be glued.
+	if !isMDFile(relDest) && !isImgFile(relDest) && r.glueLink != "" {
```

Comment:
nit: We could try to check for `r.glueLink` first as it's the  most "important" one.

Reply:
Done!